packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/delete/translations/Messages_fr_FR.json

```
@@ -1,8 +1,8 @@
 {
   "pci_databases_service_integration_delete_confirm": "Confirmer",
   "pci_databases_service_integration_delete_cancel": "Annuler",
-  "pci_databases_service_integration_delete": "Supprimer un service {{engineName}}",
-  "pci_databases_service_integration_delete_description": "Vous allez supprimer votre service {{engineName}}. Souhaitez-vous continuer ?",
-  "pci_databases_service_integration_delete_success_message": "Votre service {{engineName}} {{service}} a été supprimé avec succès",
+  "pci_databases_service_integration_delete": "Supprimer une intégration",
+  "pci_databases_service_integration_delete_description": "Vous allez supprimer votre intégration {{integrationType}}. Souhaitez-vous continuer ?",
+  "pci_databases_service_integration_delete_success_message": "Votre service intégration {{integrationType}} a été supprimée avec succès",
```

Sounds like it's the service that has been deleted, not the integration, right ?
```suggestion
  "pci_databases_service_integration_delete_success_message": "Votre service intégration {{integrationType}} a été supprimé avec succès",
```

---

Actually it is the integration that is removed, not the service. I removed the word "service"; I think it was a bad copy/paste.